skyline-vscode/react-ui/src/sections/DeploymentTab.js

```diff
@@ -32,7 +32,7 @@ const DeploymentTab = ({ numIterations, habitatData,additionalProviders }) => {
             <ProviderPanel
               numIterations={numIterations}
               habitatData={habitatData}
-              additionalProviders={additionalProviders}
+              cloudProviderURLs={additionalProviders}
```

Comment:
Can you update additionalProviders to be cloudProviderURLs as well?

Reply:
done